CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## Stash v1.0.0 Changelog
+
+### v1.0.0
+
+* Removed deprecated DriverList::getDrivers function.
+
+* Removed deprecated invalidation constants in the Item class.
+
+* Removed SQLite Extension support (SQLite3 is still available).
+
+* The `set` function no longer persists data.
+
+* Removed expiration time for `set` function
+
+* Added `expiresAt` and `expiresAfter` functions to the Item class.
+
+* `getExpiration` to return current datetime when no record exists.
```

Comment:
I don't have a problem making this return null- it originally returned false, but when getExpiration was added to the PSR they had their version return the current datetime. Now that it's been removed from the standard we can handle it however we want, so I'll put it back to null.